aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -388,8 +388,11 @@ open class AztecText : AppCompatEditText, TextWatcher, UnknownHtmlSpan.OnUnknown
 
         commentsVisible = styles.getBoolean(R.styleable.AztecText_commentsVisible, commentsVisible)
 
-        verticalParagraphMargin = styles.getDimensionPixelSize(R.styleable.AztecText_blockVerticalPadding, 0)
-        verticalHeadingMargin = styles.getDimensionPixelSize(R.styleable.AztecText_headingVerticalPadding, 0)
+
```

Comment:
```suggestion
```

It looks like the continuous build server is complaining about this extra line.  Removing it should get the build to pass.